package.json

```diff
@@ -50,6 +50,8 @@
   },
   "devDependencies": {
     "@akarui/aoi.music": "^1.2.5",
+    "discord-canvas": "^1.4.1",
+    "moment": "^2.29.3",
```

Comment:
Isn't there a better approach?

Reply:
oh wait that was for something i was testing on my bot whoops

Reply:
tho i am pretty sure moment is used by $formatDate

Reply:
I would check again.